src/frontends/pytorch/src/frontend.cpp

```diff
@@ -119,8 +119,20 @@ FrontEnd::FrontEnd() {
     }
 }
 
+std::shared_ptr<Model> FrontEnd::convert_denormalized(const ov::frontend::InputModel::Ptr& model) const {
+    FRONT_END_GENERAL_CHECK(std::dynamic_pointer_cast<pytorch::InputModel>(model), "Invalid input model");
+    try {
+        TranslateSession translate_session(model, m_op_translators, m_telemetry);
+        return translate_session.get_converted_model();
+    } catch (const std::runtime_error& e) {
+        std::cerr << "[ ERROR ] Unexpected error while converting pytorch model: " << e.what() << '\n';
+        std::cerr << "Rethrowing.\n";
```

Comment:
Can we avoid printing any messages? Being integrated into `torch.compile` it may frequently print such messages.

Reply:
okay, can delete that